### PR TITLE
This is my homework

### PR DESCRIPTION
**Pull request by Emileen**

### CODE REVIEW

**Comment by Emileen** [comment]
updates

**Comment by Emileen** [comment]
final

File: src/com/theironyard/charlotte/Bird.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package com.theironyard.charlotte;
+
+/**
+ * Created by loydcharles on 11/30/16.
+ */
+public class Bird {
+
+    String type = "peacook";
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/Bird.java]: These fields should be private.

Comment (raynjamin) [inline src/com/theironyard/charlotte/Bird.java]: Also it is a little weird that they're being initialized like this, right on the same line as the declaration. usually they'd be set in the constructor.

File: src/com/theironyard/charlotte/Car.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
+package com.theironyard.charlotte;
+
+/**
+ * Created by loydcharles on 11/30/16.
+ */
+public class Car {
+    String make;
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/Car.java]: Should be private.

File: src/com/theironyard/charlotte/Car.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    public int getYear() {
+        return year;
+
+    }
+
+    public void setYear(int carYear) {
+        year = carYear;
+        if (year < 2000) {
+            System.out.println("you car is ancient ");
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/Car.java]: lol

File: src/com/theironyard/charlotte/Car.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        } else if (year > 2000 && year < 20010) {
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/Car.java]: Uhoh! That's 20010, not 2010. (small irritating bugs later on!)

File: src/com/theironyard/charlotte/Car.java [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            System.out.println("your car is pretty new");
+        } else {
+            System.out.println("you have a sweet ride");
+        }
+    }
+
+    public double getCost() {
+        return cost;
+
+
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/Car.java]: Odd spacing here.

File: src/com/theironyard/charlotte/CustomMath.java [hunk 6]
```diff
@@ -0,0 +1,21 @@
+package com.theironyard.charlotte;
+
+/**
+ * Created by loydcharles on 11/30/16.
+ */
+public class CustomMath {
+
+
+    public static double circleArea(double a) {
+        double area=0.00;
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/CustomMath.java]: no real need to set area here to 0.

Comment (raynjamin) [inline src/com/theironyard/charlotte/CustomMath.java]: What is "a"? the radius? why not "r" or "radius?"

File: src/com/theironyard/charlotte/Dessert.java [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    void myDessert() {
+
+        System.out.println("I like " + flavour + type + " " + "and I like to eat it " + quantity + " at a time");
+
+        if (quantity > 1 && quantity < 2) {
+            System.out.println("you have self control");
+        } else {
+            System.out.println("you like your dessert");
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/Dessert.java]: I do :(

File: src/com/theironyard/charlotte/Main.java [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+import java.util.concurrent.Callable;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Person me = new Person("Emileen", "me", 35);
+        me.myInfo();
+        System.out.println();
```
Comment (raynjamin) [inline src/com/theironyard/charlotte/Main.java]: are you doing this to print new lines? This is interesting.

**Pull request closed without merging**